templates/necto/project_templates/0_mikrosdk2/CMakeLists.txt

```diff
@@ -21,6 +21,8 @@ target_link_libraries(${PROJECT_NAME}
 )
 
 if (${MIKROSDK_TYPE} STREQUAL "legacy")
+    # TODO - Temporary fix, remove with NECTO release 7.0.4+
```

Comment:
Append the same change to other templates.

I believe following have this as well:

- `library_with_example`
- `library_sdk`
- `application_with_library`

Just make sure to check which template is used with NECTO version lower than 7. That template does not need to be updated.